static/shared/package.json

```diff
@@ -1,9 +0,0 @@
-{
-  "name": "@zulip/shared",
-  "version": "0.0.13",
-  "license": "Apache-2.0",
-  "dependencies": {
-    "katex": "^0.15.3",
-    "lodash": "^4.17.19"
-  }
-}
```

Comment:
This looks like a mistake; can you remove it?

Reply:
Removed.